Resources/Prototypes/Catalog/LatheRecipes/tools.yml

```diff
@@ -62,6 +60,17 @@
   materials:
     steel: 80
 
+- type: latheRecipe
+  id: Shovel
+  icon:
+    sprite: Objects/Tools/shovel.rsi
+    state: shovel
```

Comment:
You need to change this to `icon` instead of `shovel`.

Reply:
done.